src/components/entry/EntryDetails.js

```diff
@@ -1,4 +1,41 @@
+import { Flag, Star } from '@styled-icons/boxicons-solid'
 import { useRouteMatch } from 'react-router-dom'
+//import Image from 'react'
+import styled from 'styled-components'
+
+import Button from '../ui/Button'
+import { Tag } from '../ui/Tag'
+
+const ItalicizedText = styled.p`
+  font-size: 14px;
+  font-style: italic;
+`
+
+// Just make width 100% and the height will automatically be fit
+const ImageContainer = styled.img`
+  width: 100%;
+`
+
+const TextContainer = styled.div`
+  margin: 23px;
+`
+
+const EntryHeader = styled.div`
+  line-height: 12px;
+`
+
+/* Row gap nor line-height is working here to make the distance between all elements 14px
+const DescriptionContainer = styled.div`
+  grid-template-rows: 100%
+  row-gap: 14px;
+`
+*/
```

Comment:
Talk to @SirajChokshi or @laurenho025 about the best way to achieve this. One way is to use a CSS selector to select all immediate children: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors. This might not be necessary though